src/features/artist/pages/Artist.tsx

```diff
@@ -29,7 +28,8 @@ export const Artist: FunctionComponent = () => {
     )
   }, [artistPlaylist, artistTopOffers, params.id])
 
-  if (!artist) return <PageNotFound />
+  // TODO(PC-35430): replace null by PageNotFound when wipArtistPage FF deleted
+  if (!artist) return null
```

Comment:
```suggestion
  if (!artist || !enableArtistPage) return null
```
Quitte à faire le early-return, autant regrouper les deux conditions